app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -48,111 +114,131 @@ fun TopicListScreen(modifier: Modifier = Modifier, onClickToDetail: (id : String
             }
         }
+    }
+}
 
-        Button(
-            onClick = { /*isVisible = isVisible.not()*/
-                onClickToDetail(index)
-            },
-            modifier = modifier.align(Alignment.BottomCenter)
-        ) {
-            Text("카드 선택")
-        }
+@Composable
+fun SelectBtn(viewCnt: Int, onCardClicked: () -> Unit, modifier: Modifier) {
+    Button(
+        colors = ButtonDefaults.buttonColors(backgroundColor = Color.Black),
+        onClick = {
+            if (viewCnt >= 3)
+            //TODO : 광고 시청 후 디테일로 넘어가는 로직
+                onCardClicked()
+            else
+                onCardClicked()
+        },
+        modifier = modifier
+            .padding(bottom = dimensionResource(id = R.dimen.vertical_padding))
+            .padding(horizontal = dimensionResource(id = R.dimen.horizontal_padding))
+            .height(60.dp)
+            .fillMaxWidth(),
+        shape = RoundedCornerShape(8.dp),
+    ) {
+        Text(
+            text = if (viewCnt >= 3) "광고 보고 뽑기" else "이 카드 뽑기",
+            style = TalkbbokkiTypography.button_large,
+            color = Color.White
+        )
     }
 }
 
 @Composable
-fun CardItems(definiteIndex: Int, currentOffset: Float) {
+fun CardItem(definiteIndex: Int, currentOffset: Float) {
     val pageOffsetWithSign = definiteIndex - currentOffset
     val pageOffset = pageOffsetWithSign.absoluteValue
-    Card(
-        modifier = Modifier
-            .width(120.dp)
-            .aspectRatio(0.7f)
-            .zIndex(5f - pageOffset)
-            .graphicsLayer {
-                // 중간으로 올수록 1.6배 크게 보임
-                lerp(
-                    start = 1f.dp,
-                    stop = 1.6f.dp,
-                    fraction = 1f - pageOffset.coerceIn(0f, 1f)
-                ).let { scale ->
-                    scaleX = scale.value
-                    scaleY = scale.value
-                }
+    Box(modifier = Modifier
+        .width(dimensionResource(id = R.dimen.card_width))
+        .aspectRatio(0.7f)
+        .background(Color.Transparent)
+        .zIndex(5f - pageOffset)
+        .graphicsLayer {
+            // 중간으로 올수록 크게 보임
+            lerp(
+                start = 1f.dp, stop = 1.9.dp, fraction = 1f - pageOffset.coerceIn(0f, 1.3f)
+            ).let { scale ->
+                scaleX = scale.value
+                scaleY = scale.value
+            }
 
-                // 중간으로 올수록 0도에 가까워짐. (카드 사이 각도 10도씩 틀어짐)
-                lerp(
-                    start = pageOffsetWithSign * 10f.dp, // -20, -10, 0, 10, 20 ...
-                    stop = 0f.dp,
-                    fraction = 1f - pageOffset.coerceIn(0f, 1f)
-                ).value.let { angle ->
-                    rotationZ = angle
-                }
+            // 중간으로 올수록 0도에 가까워짐. (카드 사이 각도 15도씩 틀어짐)
+            lerp(
+                start = pageOffsetWithSign * 15f.dp, // -30, -15, 0, 15, 30 ...
+                stop = 0f.dp, fraction = 1f - pageOffset.coerceIn(0f, 1f)
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)